client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -48,7 +48,7 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
           <TextField
             id="email"
-            label={<Typography className={classes.label}>E-mail address</Typography>}
+            label={<Typography className={classes.label}>Email</Typography>}
```

Comment:
It looks like yoou'll have to change something about these Text fields to match the mocks. Check out `variant="outlined"` on material ui - you'll want borders for these fields :) 

https://material-ui.com/api/text-field/#textfield-api